test/test_web.rb

```diff
@@ -663,4 +663,46 @@ def app
       assert_equal 'nicehost.org', session_options[:host]
     end
   end
+
+  describe 'sidekiq web sessions options' do
+    include Rack::Test::Methods
+
+    def app
+      Sidekiq::Web.new
+    end
+
+    after { Sidekiq::Web.enable(:sessions) }
```

Comment:
No, you are right.  Compatibility is #1.  But that doesn't mean we can't have instance methods for testing.